components/setup/BuildInsruction.tsx

```diff
@@ -44,9 +56,10 @@ const BuildInstruction: React.FC<BuildInstructionProps> = ({ selectedHosting, us
         }
     };
 
-    const renderDockerInstructions = () => {
+    const renderDockerInstructions = ({selectedInstallationType, selectedProvider}: RenderDockerInstructionsProps) => {
```

Comment:
Low LOGAF.

It looks like this file is getting too big. We should probably make different components for each part and maybe pass the details as props. 

You can pick this up for the next PR. Shouldn't be a big change.

Reply:
Yeah I was thinking the same.